App.js

```diff
@@ -27,32 +29,49 @@ const App = () => {
     const parsedRes = await res.json();
     setData(parsedRes);
   };
+
   const [data, setData] = useState();
 
   useEffect(() => {
     fetchData();
   }, []);
 
-  console.log(data);
+   //Function used by flatlist that indicates how the item is going to be seen
+  const renderItem =  ( {item} ) => (
```

Comment:
With the code style settings that we have in this project the spacing for `( {item} )` should be `({item})`, in the actual effectus project app it will be `({ item })`.

Sorry for the confusion, perhaps it's worth importing those configs to this prototype 🤔